impl/src/main/java/org/ehcache/config/builders/UserManagedCacheBuilder.java

```diff
@@ -80,10 +79,10 @@
 import static org.ehcache.config.ResourceType.Core.OFFHEAP;
 import static org.ehcache.config.builders.ResourcePoolsBuilder.newResourcePoolsBuilder;
 import static org.ehcache.core.internal.service.ServiceLocator.dependencySet;
+import static org.ehcache.core.internal.service.ServiceLocator.findSingletonAmongst;
```

Comment:
Yes. I noticed. But they are in the right other now. Do you prefer to leave them unordered?